moviesnetwork/moviesdata/services.py

```diff
@@ -0,0 +1,12 @@
+import imdb
+
+from data import Actor
+
+
+def search_actor(name):
+    imdb_access = imdb.IMDb()
+    result = imdb_access.search_person(name)
+    actors = []
+    if result:
+        actors = [Actor.create(r, False) for r in result]
```

Comment:
`actors = [Actor.create(r, False) for r in result] if result else []` sounds more pythonic